app/scrapers/sections_scraper.rb

```diff
@@ -77,12 +84,20 @@ def parse_sections(url, semester)
 logger = ScraperCommon::logger
 
 semesters = ScraperCommon::get_semesters(ARGV)
+# course id accumulator. Flushed each time a sections GET request is made.
+# @type [Array<Number>]
 courses = []
 
 # Loop through the semesters we want to parse
 semesters.each do |semester|
   # Arrays to hold the things we want to insert
   sections = []
+  # @param [Array<String | Number>] courses
+  # @return [String]
+  def make_query courses
```

Comment:
Where is this used?

Reply:
trying to DRY up the code, haven't actually started using it. Don't want to do *too* much in one PR.

Reply:
I'm not sure I'm a fan of adding a method in a pr without using it...without seeing how it's being used there's basically no way for us to consider whether this is a good method to have or a good way for it to be written

Reply:
I mean just look at the strings below that function def. I can start using it, should be fine.

Reply:
if it's that simple then might as well start using it yeah? I just don't see the point in splitting it over prs